bmrg-flow/values.yaml

```diff
@@ -163,9 +162,10 @@ monitoring:
       host: jaeger-dev-agent
       port: 5778
 
-jetstream:
-  enabled: false
-  url: nats://localhost:4222
+# Eventing related values
+eventing:
+  enabled: true
```

Comment:
We don't by default enable eventing. Can we switch this to `false`

Reply:
Fixed.